test/alphabet/nucleotide/CMakeLists.txt

```diff
@@ -1,15 +1,15 @@
 add_executable(dna5_test dna5_test.cpp)
```

Comment:
Since #44 contains all the changes to this file already (and more changes), can you just skip this file?